doc/adr/0016-use-spring-mvc-instead-of-webflux.md.md

```diff
@@ -0,0 +1,85 @@
+# 16. Use Spring MVC instead of WebFlux
+
+Date: 2021-12-22
+
+## Status
+
+Accepted
+
+## Context
+
+The team continues to struggle with getting comfortable & confident with WebFlux and reactive programming.
+
+### Cognitive (over-)load
+
+Due to the lack of team experience with the framework and paradigm, a significant learning effort is required. Learning reactive programming and the WebFlux APIs contributes to team already high cognitive load.
+
+The team has to navigate a complex project context:
+- Complex political stakeholder landscape → short planning horizons, frequently shifting vision and goals.
+- Developers need to do participate in frequent product discovery work, incl. understanding where we fit into the existing landscape of technology & service providers.
+- Direct engagement of developers with sophisticated technical stakeholders: BSI, CCC, D-Trust, Adesso, etc.
+- Complex landscape of domain-specific technical requirements: eID Technische Richtlinien, EIDAS, NKB, etc.
+- Limited team experience with hosting platform: ArgoCD, Terraform, Kubernetes.
+- Limited team experience with the application stack: Kotlin, Spring, WebFlux.
```

Comment:
```suggestion
- Limited team experience with the application stack: Kotlin, Spring.
```

Reply:
I'd leave this in, since the text describes the status quo before removing WebFlux?

Reply:
Ok 👍